docs/digital_forensic_research_workshop.md

```diff
@@ -1,56 +1,115 @@
+One focus of the CDO project is Cyber-investigation Analysis Standard 
+Expression (CASE) which was formally launched at DFRWS EU 2015 and 
+presented as “Leveraging CybOX to Standardize Representation and 
+Exchange of Digital Forensic Information.”
 
-## Sponsorships
+Another DFRWS supported project called Systematic Objective-based 
+Listing of Various Established [Digital] Investigation Techniques 
+(SOLVE-IT) to establish a resource and tools for systematic error 
+mitigation for digital evidence. This initiative was started by the 
+community at DFRWS US 2023, and formally launched at DFRWS EU 2025 
+and presented as “SOLVE-IT: A proposed digital forensic knowledge 
+base inspired by MITRE ATT&CK.”
 
-Sponsors of the conference will gain visibility for their companies,
+## Support and Sponsorship
+
+Sponsors of the conference gain visibility for their companies,
 demonstrate their support of forensic research and development, and
 contribute to the success of the conference. All sponsors will have
-their logo listed on the DFRWS 2008 website with a link to their website
+their logo listed on the DFRWS website with a link to their website
 and will be listed in the printed proceedings. They will also get
 recognition at specific events that they sponsor.
 
-## DFRWS Challenges
-
-## 2008
+DFRWS is also supported and extended by Satellite Sites hosted by an 
+institution that plans an event which incorporates a portion of the 
+conference stream, allowing a group to participate in the conference 
+in their region. These Satellite Sites increase opportunities for 
+members of the growing community to meet and exchange knowledge.
 
-The [DFRWS 2008 Challenge](http://old.dfrws.org/2008/challenge/) focuses on the
-development of Linux memory analysis techniques and the fusion of evidence from
-memory, hard disk, and network. Since the DFRWS 2005 Challenge, there has been
-significant progress in Windows memory analysis. Now, we are focusing on Linux
-and on integrating evidence from multiple sources.
+## DFRWS Forensic Challenges
 
-### 2017 - 2018
+Since 2005, DFRWS has put forth challenge scenarios with associated 
+data created by volunteer conference organizers to advance research 
+and development  in emerging areas of Digital Forensics. These 
+challenges have spawned new specializations and capabilities, 
+including Windows Memory Forensics, Linux Memory Forensics, File 
+Carving, Mobile Device Forensics, Malware Forensics, and IoT Forensics. 
+Anyone can submit their analysis and tools that they created for a DFRWS 
+Forensic Challenge. Winners are chosen by a committee and prizes include 
+free registration for the DFRWS conference that year and grand prizes
+for exceptional contributions to the field.
 
-The 2017 - 2018 DFRWS Challenge is about the Internet of Things (IoT).
-Analysis of devices and network data including a Smart TV Raspberry Pi
-and Amazon Echo cloud data are part of the scenario.
+| Year | Topic | Impact |
```

Comment:
these tables go out of date quickly, what about a link on the dfrws site that contains this overview?